stepmod-utils.gemspec

```diff
@@ -35,12 +35,14 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "expressir"
   spec.add_runtime_dependency "glossarist", "~> 2.0"
   spec.add_runtime_dependency "indefinite_article"
+  spec.add_runtime_dependency "plurimath"
   spec.add_runtime_dependency "ptools"
   spec.add_runtime_dependency "pubid-iso"
   spec.add_runtime_dependency "octokit"
   spec.add_runtime_dependency "down"
   spec.add_runtime_dependency "reverse_adoc", ">= 0.3"
   spec.add_runtime_dependency "thor", ">= 0.20"
+  spec.add_runtime_dependency "unitsml"
```

Comment:
I think Plurimath already includes `unitsml` so we don't need it here?

Reply:
@ronaldtse Currently `unitsml` require `plurimath` so if we add `unitsml` as a dependency in `plurimath` it creates a circular dependency so it is not included in `plurimath` in the latest version.